js/client-teambuilder.js

```diff
@@ -2376,7 +2376,8 @@
 				this.search.find('');
 				return;
 			}
-			var val = $(e.currentTarget).data('entry').split(':')[1];
+			var entry = $(e.currentTarget).data('entry');
+			var val = $(e.currentTarget).data('entry').slice(entry.indexOf(":") + 1);
```

Comment:
...you know, when I said "holy duplicate code", I meant that this should be

    entry.slice(entry.indexOf(":") + 1)

I think you missed the "duplicate code" part.